### ISSUE TITLE
Code review (accessibility)

### ISSUE THREAD

**Issue by BenLubar** [Chocobozzz/PeerTube#2149]
(bugs marked with 🐛)

https://github.com/Chocobozzz/PeerTube/blob/9a18a6252071cf21b18f82a24bb63078abb75bc1/client/src/app/shared/video/video-thumbnail.component.html#L5

🐛 This should be using `aria-label` as the video title is not the ID of an element. Alternatively, since another nearby UI element already has the same action and label, just set it to `aria-hidden="true"` so screen readers don't say the name twice.

https://github.com/Chocobozzz/PeerTube/blob/1198edf4bb06ce5f1668b97cf9ca8fb483fe3f41/client/src/index.html#L1

This should have a `lang` attribute equal to the current language. Otherwise, screen readers will assume an arbitrary language (likely the system default) which could be wrong.

https://github.com/Chocobozzz/PeerTube/blob/457bb213b273a9b206cc5654eb085cede4e916ad/client/src/app/header/header.component.html#L2

Needs an `aria-label` attribute (something like "video search" or "search" would be fine)

[pretty much every use of `my-global-icon`]

Should have an `aria-label` attribute, and has a weird ➕-shaped hitbox in the browser if it's taller than the nearby text.

[wherever the glyphicons font-face is defined]

🐛 Should have `font-display: swap;` to avoid browsers blocking on this file.

[wherever link color is defined in css]

🐛 `#007ad9` is too dark to be easily readable in the dark theme - this should be changed to use a variable.

https://github.com/Chocobozzz/PeerTube/blob/5819e694ba238b5e4ee97347c6cf5ee7a82725b5/client/src/sass/include/_variables.scss#L9-L12

🐛 This shade of grey cannot easily be read on a dark background - this should be changed to use a variable.

https://github.com/Chocobozzz/PeerTube/blob/bfbd912886eba17b4aa9a40dcef2fddc685d85bf/client/src/app/videos/%2Bvideo-watch/video-watch-playlist.component.scss#L16

🐛 This makes playlist titles unreadable on the dark theme.

https://github.com/Chocobozzz/PeerTube/blob/0ba5f5baade94a051710d9d9d408ac8083c14ac6/client/src/app/%2Bmy-account/my-account-settings/my-account-change-password/my-account-change-password.component.html#L6-L25

These (along with other `input type="password"` elements) should have `autocomplete` attributes. `current-password` for fields that take the user's current password, and `new-password` for fields that change the user's password. Also consider moving the password and email changing forms to another page so password managers don't prompt the user to enter their password when they're just editing their profile data.

**Comment by LeaveToDream** [comment]
Hi, i'm interested in working on this :)

**Comment by Chocobozzz** [comment]
@LeaveToDream I don't think adding `aria-label`  in these cases is necessary.

**Comment by Chocobozzz** [comment]
Any news @LeaveToDream? :)

**Comment by rigelk** [comment]
this was mostly implemented in 7738273.

**Comment by LeaveToDream** [comment]
I forgot to finish and push, and then time fled. I'm deeply sorry about that.
